src/lls_model.jl

```diff
@@ -69,3 +81,83 @@ function hprod_residual!(nls :: LLSModel, x :: AbstractVector, i :: Int, v :: Ab
   fill!(Hiv, 0.0)
   return Hiv
 end
+
+function cons(nls :: LLSModel, x :: Vector)
+  increment!(nls, :neval_cons)
+  return nls.C * x
+end
+
+function cons!(nls :: LLSModel, x :: Vector, c :: Vector)
+  increment!(nls, :neval_cons)
+  c[1:nls.meta.ncon] = nls.C * x
+  return c
+end
+
+function jac_coord(nls :: LLSModel, x :: Vector)
+  increment!(nls, :neval_jac)
+  if isa(nls.C, AbstractMatrix)
+    return findnz(J)
+  else
+    return findnz(full(J))
```

Comment:
Is that really what we want? Linear operators should probably not be used that way. If constraints are defined by an operator, it means we shouldn't perform matrix operations.

Reply:
You're right, that makes more sense.